src/screens/LocationPage/LocationPage.js

```diff
@@ -10,6 +11,30 @@ import EnsureSharingIdInUrl from 'components/EnsureSharingIdInUrl';
 import ChartsHolder from 'components/LocationPage/ChartsHolder';
 import { LoadingScreen } from './LocationPage.style';
 import { useProjections } from 'common/utils/model';
+import { findLocationForFips, findStateByFips } from 'common/locations';
+import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
+
+function getPageTitle(projections) {
+  const location = findLocationForFips(projections.fips);
+  const state = findStateByFips(location.state_fips_code);
+  const locationName = location.county
+    ? `${location.county}, ${state.state}`
+    : `${location.state}`;
+  return `${locationName}, ${location.state_code} - America’s COVID Warning System`;
+}
+
+function getPageDescription(projections) {
+  const dateToday = moment().format('MMM DD, YYYY');
+  const location = findLocationForFips(projections.fips);
+  const state = findStateByFips(location.state_fips_code);
+  const alarmLevel = projections.getAlarmLevel();
+  const levelInfo = LOCATION_SUMMARY_LEVELS[alarmLevel];
+  const locationName = location.county
+    ? `${location.county}, ${state.state}/${location.state_code}`
```

Comment:
Again, instead of like "Blah, New York/NY" I think "Blah, New York (NY)" would be better.